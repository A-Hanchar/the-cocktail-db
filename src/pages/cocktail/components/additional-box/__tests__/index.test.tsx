import {render} from '@testing-library/react';

import {AdditionalBox} from '../';

describe('<AdditionalBox />', () => {
	it('SHOULD match snapshot', () => {
		const {asFragment} = render(<AdditionalBox title='title-mock' />);

		expect(asFragment()).toMatchSnapshot();
	});
});
