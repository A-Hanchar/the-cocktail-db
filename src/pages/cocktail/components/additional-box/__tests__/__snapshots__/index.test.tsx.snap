// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AdditionalBox /> SHOULD match snapshot 1`] = `
<DocumentFragment>
  <div
    class="box additionalBox"
    data-test-id="box"
  >
    <h1
      class="ui-text color-base size-36px"
      data-test-id="text"
    >
      title-mock
    </h1>
  </div>
</DocumentFragment>
`;
